### PR TITLE
Adopt OctoLight

### LINKED ISSUE #1225: [Abandoned Plugin] OctoLight
### Which Plugin?

https://plugins.octoprint.org/plugins/octolight/

### Link to ticket on plugin's issue tracker confirming abandonment

https://github.com/gigibu5/OctoLight/issues/35

### Is there someone open to adopting it?

_No response_

### PR DESCRIPTION
**Pull request by thomst08**
## New maintainer of OctoLight

Closes: #1225 

An issue was created on the original OctoLight repo to check if it was abandoned. I showed interest in maintaining this plugin, @gigibu5 was happy for me to take over.
https://github.com/gigibu5/OctoLight/issues/41

Please let me know if I have done anything incorrectly or need to update anything else.

### CODE REVIEW

File: _plugins/octolight.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ id: octolight
 title: OctoLight
 description: A simple plugin, that add's a button to the navbar, toggleing GPIO on the RPi. It can be used for turning on and off a light.
 authors:
-- Žiga Kralj
+- Steven Thomson, previously Žiga Kralj
```
Comment (jneilliii) [inline _plugins/octolight.md]: prefer to use this formwat

```
authors:
- Steven Thomson
- Žiga Kralj
```

File: _plugins/octolight.md [hunk 2]
```diff
@@ -81,4 +81,28 @@ compatibility:
 
 ---
 
-A simple plugin that adds a button to the navigation bar for toggleing a GPIO pin on the Raspberry Pi. I use it for turning ON and OFF the light on my 3D printer.
+A simple plugin that adds a button to the navigation bar for toggling a GPIO pin on the Raspberry Pi. This plugin has the function to turn on or off based on a printer event or manually through a user interaction. Printer events can also trigger a light turn off after a specified time.
+
+![WebUI interface](img/screenshoot.png)
```
Comment (jneilliii) [inline _plugins/octolight.md]: url should be `/assets/img/plugins/octolight/screenshot.png`

File: _plugins/octolight.md [hunk 3]
```diff
@@ -81,4 +81,28 @@ compatibility:
...
+
+
+## Configuration
+![Settings panel](img/settings.png)
```
Comment (jneilliii) [inline _plugins/octolight.md]: url should be `/assets/img/plugins/octolight/settings.png`

**Review by jneilliii — COMMENTED** [review]
couple of small tweaks necessary. I notice heavy usage of self._logger.info in your code. might recommend most those be switched to self._logger.debug to avoid writing too much to the octoprint.log.

File: _plugins/octolight.md (hunk 2 above)
Comment (thomst08) [reply _plugins/octolight.md]: Sorry, I should have checked this, this has been corrected

**Comment by thomst08** [comment]
Thank you regarding the point about the logs, I will go ahead and update the code shortly :)

**Review by jneilliii — APPROVED** [review]

**Comment by ArtSpaziano** [comment]
Is it possible to add another Event option to have the light go on when a filament change occurs?

**Comment by thomst08** [comment]
Hey @ArtSpaziano,

Submit an issue on my repo and I can look into it, we can discuss it more there. 

https://github.com/thomst08/OctoLight

**Pull request merged**